debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java

```diff
@@ -247,6 +250,14 @@
         return fields;
     }
 
+    protected List<SchemaAndValueField> schemaAndValueForStringEncodedHStoreType(){
+        return Arrays.asList(new SchemaAndValueField("hs",Json.builder(),"\"key\" => \"val\""));
```

Comment:
The expected value must be given as a JSON document here, i.e. something like `"{ \"key\" : \"value\" }"`.